api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -106,6 +108,34 @@ constructor(
         ), egressEvent = dbEvent, escalation = escalation)
     }
 
+    override fun fireAdminEditEgressEvent(
+        previousEvent: DbEgressEvent,
+        updatedEvent: DbEgressEvent
+    ) {
+        val changesByProperty = TargetPropertyExtractor.getChangedValuesByName(
+                DbEgressEventTargetProperty.values(),
+                previousEvent,
+                updatedEvent)
+        val dbWorkspaceMaybe = Optional.ofNullable(updatedEvent.workspace)
+        val actionId = actionIdProvider.get()
+        val admin = userProvider.get()
+        actionAuditService.send(changesByProperty.entries.map {
+            ActionAuditEvent(
+                    timestamp = clock.millis(),
+                    actionId = actionId,
+                    actionType = ActionType.EDIT,
+                    agentType = AgentType.ADMINISTRATOR,
+                    agentIdMaybe = admin.userId,
+                    agentEmailMaybe = admin.username,
+                    targetType = TargetType.WORKSPACE,
```

Comment:
`TargetType.WORKSPACE i`s not 100% accurate, but I guess that is the most closest one.

Reply:
Yeah.. agreed. Possibly we can consider changing it later, though for now it is nice that the egress alert history will show up on the workspace audit page automatically.

This is mostly following from the original convention that an egress event firing is attached to a workspace.